mappings/net/minecraft/data/client/model/ModelIds.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_4941 net/minecraft/data/client/model/ModelIds
+	METHOD method_25840 getItemModelId (Lnet/minecraft/class_1792;)Lnet/minecraft/class_2960;
+		ARG 0 item
+	METHOD method_25841 getItemSubModelId (Lnet/minecraft/class_1792;Ljava/lang/String;)Lnet/minecraft/class_2960;
+		ARG 0 item
+		ARG 1 suffix
+	METHOD method_25842 getBlockModelId (Lnet/minecraft/class_2248;)Lnet/minecraft/class_2960;
+		ARG 0 block
+	METHOD method_25843 getBlockSubModelId (Lnet/minecraft/class_2248;Ljava/lang/String;)Lnet/minecraft/class_2960;
+		ARG 0 block
+		ARG 1 suffix
+	METHOD method_25844 getMinecraftBlock (Ljava/lang/String;)Lnet/minecraft/class_2960;
+		ARG 0 name
+	METHOD method_25845 getMinecraftItem (Ljava/lang/String;)Lnet/minecraft/class_2960;
+		ARG 0 name
```

Comment:
Why specifying "Minecraft" here?

Reply:
Because this is getting model id of a "minecraft" block/item (i.e. a block/item with "minecraft" namespace)

Reply:
Then what about `getDefaultNamespaceBlock/Item`?

Reply:
I recall the id in these places are constructed explicitly by `new Identifier("minecraft", name)` and these are deprecated (in favor of versions taking regular `Identifier`s)

Reply:
Using "Minecraft" doesn't make it clear that it's about the vanilla/default namespace. It just makes it clear that it's in Minecraft game, which is pointless

Reply:
The namespace is called `minecraft`. If the literal is `vanilla`, we use vanilla, like `VanillaDataPackProvider`. Also when we say "minecraft item" in modded context we usually know it's something from "minecraft" mod than some other mod. @Runemoro mind discuss?

Reply:
I agree with @Yanis48. All items are Minecraft items, but not all items are "`minecraft:`" items. The best way to make it clear in the name is to call this `getVanillaItem`.

Reply:
Should this be `getMinecraftModItem` as this gets an item from the "minecraft" mod? there is no mod called `vanilla`.

Reply:
`getMinecraftNamespaceItem`. We shouldn't call Minecraft a mod (mod = modification to the vanilla game).

Reply:
Though I would prefer `getVanillaItem`. I don't see why that name would make someone think it gets a `vanilla:` namespace item...

Reply:
I think `getVanillaItem` makes more sense because mojang refers to certain things as "Vanilla", such as `VanillaLayeredBiomeSource`